index.html

```diff
@@ -0,0 +1,874 @@
+                <h6><a class="nav-link text-nowrap" href="#">Vantage <span class="large text-muted"> | </span><span class="short"> > </span></a></h6>
+              </li>
+              <li class="nav-item">
+                <h6><a class="nav-link text-nowrap" href="#">Weather <span class="short"> > </span></a></h6>
+              </li>
+        </ul>
+        <form class="form-inline my-2 my-lg-0 input-group d-none d-lg-flex">
+          <input class="form-control mr-sm-2 input-group-text" type="search" placeholder="Search" aria-label="Search">
+          <button class="btn my-2 my-sm-0 input-group-append" type="submit"><i class="material-icons">search</i></button>
+        </form>
+      </div>
+    </nav>
+    <!-- Main section  -->
+  <main class="grid-area bg-white">
+      <!-- Top story -->
+      <article class="top-story mb-4 bg-white">
+        <h5 class="text-muted text-uppercase">Top Story</h5>
+        <img class="img-fluid" src="img/bootstrap-top.jpg" alt="Bootstrap">
+        <h2 class="text-weight-bold">Bootstrap makes building a site faster, developers say</h2>
+        <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. </p>
+      </article>
+      <!-- Featured stories -->
+      <article class="featured-stories bg-white">
+        <h5 class="text-muted text-uppercase">Featured Stories</h5>
+        <img class="img-fluid" src="img/image-1.png" alt="Image 1">
+        <h2 class="text-weight-bold"><a href="#">Web design never was so easy</a></h2>
+        <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. </p>
+
+        <img class="img-fluid" src="img/image-2.png" alt="Image 2">
+        <h2 class="text-weight-bold"><a href="#">Is Bootstrap 4 the future framework?</a></h2>
+        <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. </p>
+
+        <img class="img-fluid feat-extra" src="img/image-2.png" alt="Image 2">
+        <h2 class="text-weight-bold feat-extra"><a href="#">Is Bootstrap 4 the future framework?</a></h2>
+        <p class="feat-extra">Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. </p>
+      </article>
+      <!-- Opinion section-->
+      <article class="opinion-section bg-white">
+        <h5 class="text-muted text-uppercase">Opinion</h5>
+        <section class="d-flex">
```

Comment:
- [ ]  The images in this section are larger in the medium and small breakpoints. Also, check the font-size and adjust it.